services/event/tests/event_test.go

```diff
@@ -430,7 +430,7 @@ func TestMarkUserAsAttendingEventService(t *testing.T) {
 
 	expected_event_tracker := models.EventTracker{
 		EventID: "testid",
-		Users:   []string{"testuser"},
+		Users:   map[string]int{"testuser": int(time.Now().Unix())},
```

Comment:
It looks like these test may be flaky - if they run on the boundary of a particular second, the time in the expected event may not be identical to the time in the event you create.

I would recommend allowing some tolerance in the time associated with a user's check-in. You may need to use something other than `DeepEqual` in this case.